src/desktop/apps/article/components/layouts/__tests__/Classic.jest.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { ClassicArticleLayout } from "../Classic"
+import { mount } from "enzyme"
+import {
+  ClassicArticle,
+  ClassicArticleInternalChannel,
+  ClassicArticlePromotedContent,
+} from "@artsy/reaction/dist/Components/Publishing/Fixtures/Articles"
+jest.mock("desktop/collections/articles.coffee")
+jest.mock("desktop/components/articles_grid/view.coffee")
+const ArticlesGridView = require("desktop/components/articles_grid/view.coffee") as jest.Mock
+
+describe("ClassicArticleLayout", () => {
+  let props
+  const getWrapper = (passedProps = props) => {
+    return mount(<ClassicArticleLayout {...passedProps} />)
+  }
+
+  beforeEach(() => {
+    props = {
+      article: ClassicArticle,
+      templates: {
+        ArticlesGridView: "articles-grid-view",
+      },
+    }
+  })
+
+  it("renders an article", () => {
+    const component = getWrapper()
+    expect(component.html()).toMatch("ClassicHeader")
+    expect(component.html()).toMatch(
+      "New Study of Yale Grads Shows the Gender Pay Gap for Artists Is Not So Simple"
+    )
+    expect(component.html()).toMatch("Joanne Artman Gallery")
+  })
+
+  it("renders a team-channel article", () => {
+    props.article = ClassicArticleInternalChannel
+    const component = getWrapper()
+    expect(component.html()).toMatch("ClassicHeader")
```

Comment:
This might be a micro-optimization, but `component.html()` could be called once instead of three times and potentially speed up the tests.